doc/examples/joystick/lib/player.dart

```diff
@@ -25,14 +25,15 @@ class Player extends Component implements JoystickListener {
 
   @override
   void render(Canvas canvas) {
-    if (_rect != null) {
-      canvas.save();
-      canvas.translate(_rect.center.dx, _rect.center.dy);
-      canvas.rotate(radAngle == 0.0 ? 0.0 : radAngle + (pi / 2));
-      canvas.translate(-_rect.center.dx, -_rect.center.dy);
-      canvas.drawRect(_rect, _paint);
-      canvas.restore();
+    if (_rect == null) {
```

Comment:
this isn't changing much, any reason for this?

Reply:
This looks more consistent with the check in `moveFromAngle` below and other code around the project.

And this follows the `return early` best practice.